proposed/http-middleware/middleware-meta.md

```diff
@@ -1,49 +1,56 @@
-PSR-15 Meta Document
-====================
+HTTP Server Middleware
+======================
 
 1. Summary
 ----------
 
 The purpose of this PSR is to provide an interface that defines the formal
-method signature for HTTP Middleware that is compatible with HTTP Messages,
-as defined in PSR-7.
+method signature for HTTP Server Middleware that is compatible with HTTP
+Messages, as defined in [PSR-7][psr7].
+
+[psr7]: http://www.php-fig.org/psr/psr-7/
 
 2. Why Bother?
 --------------
 
-The general concept of reusable middleware was popularized by [StackPHP][stackphp].
-Since the release of the HTTP Messages standard, a number of frameworks have
-adopted middleware that uses HTTP Message interfaces.
+The HTTP Messages specification does not contain any reference to HTTP Middleware.
+
+The design pattern used by middleware has existed for many years as the
+[pipeline pattern][pipeline], or more specifically, "linear pipeline processing".
+The general concept of reusable middleware was popularized within PHP by
+[StackPHP][stackphp]. Since the release of the HTTP Messages standard, a number
+of frameworks have adopted middleware that use HTTP Message interfaces.
 
-Agreeing on a formal middleware interface eliminates several problems and
+Agreeing on a formal server middleware interface eliminates several problems and
 provides a number of benefits:
 
 * Provides a formal standard for middleware developers to commit to.
 * Eliminates duplication of similar interfaces defined by various frameworks.
 * Avoids minor discrepancies in method signatures.
 * Enables any middleware component to run in any compatible framework.
 
+[pipeline]: https://en.wikipedia.org/wiki/Pipeline_(computing)
 [stackphp]: http://stackphp.com/
-[express]: http://expressjs.com/en/guide/writing-middleware.html
 
 3. Scope
 --------
 
-## 3.1 Goals
+### 3.1 Goals
 
 * Create a middleware interface that uses HTTP Messages.
-* Provide a suggested interface for middleware stack containers.
-* Ensure that middleware will not be tied to a specific implementation of HTTP Messages.
+* Ensure that middleware will not be coupled to a specific implementation of HTTP Messages.
```

Comment:
I'm going to add a note to the top of this document that makes it clear that all references "middleware" mean "server middleware".